src/plugins/chart_expressions/expression_xy/public/components/xy_chart.tsx

```diff
@@ -745,6 +798,56 @@ export function XYChart({
         }}
       >
         <Chart ref={chartRef}>
+          <TooltipSpec
+            boundary={document.getElementById('app-fixed-viewport') ?? undefined}
+            headerFormatter={
+              !args.detailedTooltip
+                ? ({ value }) => (
+                    <TooltipHeader
+                      value={value}
+                      formatter={safeXAccessorLabelRenderer}
+                      xDomain={rawXDomain}
+                    />
+                  )
+                : undefined
+            }
+            actions={
+              !args.detailedTooltip
```

Comment:
ℹ️ I am enabling this only for the EC tooltip and not the custom detailed one (it is only used for agg based)